docs/wiki/EL/el-architecture.md

```diff
@@ -0,0 +1,66 @@
+# Overview 
+
+Beyond its fundamental role of transaction execution , the Execution Layer client undertakes several critical responsibilities. These include synchronizing its blockchain copy, facilitating network communication through gossip protocols with other Execution Layer clients, minting a transaction pool, and fulfilling the Consensus Layer's requirements that drive its functionality. This multifaceted operation ensures the robustness and integrity of the Ethereum network.
+
+The client's architecture is grounded in a series of detailed specifications, each playing a unique role in its comprehensive functionality. This document aims to provide a concise overview of the core specification. For a deeper understanding of how all the specifications seamlessly work together within the Execution Layer Client.
+
+# Engine  
+
+The Execution-Layer's Engine that exposes the Engine API to the consensus layer allows for running multiple instances of the execution layer but accepts only one instance of consensus layer , furthermore the engine API uses the JSON-RPC interface over multiple transports , of which HTTP and Websockets are authenticated via a [JWT](https://jwt.io/introduction) token and for ipc communication no authentication is required . Additionally the engine JSON-RPC is not exposed to anyone besides the consensus layer. However its important to note that the JWT is primarily used for authenticating the Payload , i.e. the sender is the consensus layer , it does not necessarily encrypt such traffic , thus its possible for network sniffers to observe these exchanges or attackers to perform [replay-attacks](https://en.wikipedia.org/wiki/Replay_attack)
+
+## ForkChoice Updated
```

Comment:
I assume these are headers to elaborate on?

Reply:
yes

Reply:
Got it. In that case, I would create the intro overview with a scheme you are going to follow throughout the text.